casper/src/main/scala/coop/rchain/casper/BlockDag.scala

```diff
@@ -9,7 +9,8 @@ final case class BlockDag(idToBlocks: Map[Int, BlockMessage],
                           blockLookup: Map[BlockHash, BlockMessage],
                           childMap: Map[BlockHash, Set[BlockHash]],
                           latestMessages: Map[Validator, BlockHash],
-                          currentId: Int)
+                          currentId: Int,
+                          currentSeqNum: Map[Validator, Int])
```

Comment:
So this `seqNum` is something like an Ethereum account nonce, but we only have them for validators.

Reply:
Yeah